content/pt-br/docs/reference/issues-security/issues.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Rastreador de Issue Kubernetes
+weight: 10
+aliases: [/cve/,/cves/]
```

Comment:
Hi there, @AnaCarolinaRodriguesLeite! 👋 

These aliases need to be updated to prepend `/pt-br/` to them. Would you mind addressing this?

I'll be back soon to review the remainder of this page.

Reply:
Hello Mauren Betin, how are you?
I couldn't understand very well what should be done here, could you explain?

Reply:
Hi there!

These need to be `/pt-br/cve/,/pt-br/cves/` instead of just `/cve/,/cves/`. This is because the aliases are appended to docs.k8s.io to direct directly to this page (i.e. docs.k8s.io/cves), so if we keep it like this it will end up overriding the English version, which we do not want.

Prepending `/pt-br/` will result in an URL like docs.k8s.io/pt-br/cves after merged, which doesn't override the English version.